nautobot/users/tests/test_views.py

```diff
@@ -235,14 +238,28 @@ def test_get_object_with_constrained_permission(self):
         self.assertHttpStatus(self.client.get(instance1.get_absolute_url()), 302)
 
         # Try GET to non-permitted object
-        self.assertHttpStatus(self.client.get(instance2.get_absolute_url()), 404)
+        # Should be able to get to any SavedView instance as long as the user has "{app_label}.view_{model_name}" permission
+        self.assertHttpStatus(self.client.get(instance2.get_absolute_url()), 302)
```

Comment:
Is this test now *assuming* that instance1 and instance2 reference the same list view?

Reply:
Refactored